### PR TITLE
Parallelize MTX generation (#86)

### PR DESCRIPTION
**Pull request by dsotirho-ucsc**
Ticket https://github.com/DailyDreaming/load-project/issues/86

### CODE REVIEW

File: csv2mtx.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 from pathlib import Path
 from shutil import copyfileobj
 import sys
+import threading
+# from tqdm import tqdm
```
Comment (hannes-ucsc) [inline csv2mtx.py]: -

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
Remove all changes not essential to concurrency. Associate errors in worker threads with the GEO project and report projects with any errors, those raised in the main thread as well as those raised in worker threads.

**Comment by hannes-ucsc** [comment]
Use a log formatter to add thread name to all log messages.

**Comment by dsotirho-ucsc** [comment]
Rebased onto latest master.
Assigning to @hannes-ucsc for remaining todo: handle exceptions in threads.

**Pull request merged**